clang/lib/Headers/aiev2_core.h

```diff
@@ -1566,4 +1566,300 @@ inline v16accfloat addmsc_4x8_8x4( v32float v1, v32float v2, v16accfloat acc1, v
 
 #endif
 
+enum class aie_dm_resource {
+  none,
+  PM,
+  DM,
+  DM_test,
+  stack,
+  a,
+  b,
+  c,
+  d,
+  ab,
```

Comment:
These are "a or b" etc?

Reply:
Yes "ab" are compound bank annotation, which represent the pointer could access memory in Bank A or B, etc.

Reply:
If the point is to describe memory banks, then shouldn't we be focusing only on the memory bank aspect?  I assume you're modeling it this way to get compatibility with existing code?